packages/theme-toolkit/src/component-stories-candidate.tsx

```diff
@@ -0,0 +1,377 @@
+    },
+  },
+  {
+    storyId: 'react-nl-link--visited',
+    group: STORY_GROUPS['LINK'],
+    name: 'Candidate Link: Visited',
+    state: true,
+    inline: true,
+    render: () => (
+      <Link href="https://example.com" target="_new" className="utrecht-link--visited">
+        The Quick Brown Fox Jumps Over The Lazy Dog
+      </Link>
+    ),
+    detectTokens: {
+      anyOf: ['nl.link.visited.color'],
+    },
+  },
+  {
+    storyId: 'react-nl-mark--default',
+    name: 'Candidate Mark',
+    render: () => <Mark>Mark my words!</Mark>,
+    detectTokens: {
+      anyOf: ['nl.mark.color', 'nl.mark.background-color'],
+    },
+  },
+  {
+    storyId: 'react-nl-number-badge--default',
+    group: STORY_GROUPS['BADGES'],
+    name: 'Candidate Number Badge',
+    render: () => <NumberBadge>42</NumberBadge>,
+  },
+  {
+    storyId: 'react-nl-paragraph--default',
+    group: STORY_GROUPS['PARAGRAPH'],
+    name: 'Candidate Paragraph',
+    render: () => (
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+        laborum
+      </Paragraph>
+    ),
+    detectTokens: {
+      anyOf: [
+        'nl.paragraph.color',
+        'nl.paragraph.font-size',
+        'nl.paragraph.font-family',
+        'nl.paragraph.font-weight',
+        'nl.paragraph.line-height',
+        'nl.paragraph.margin-block-start',
+        'nl.paragraph.margin-block-end',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-paragraph--lead',
+    component: 'nl-paragraph',
+    group: STORY_GROUPS['PARAGRAPH'],
+    name: 'Candidate Lead Paragraph',
+    render: () => (
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+        laborum
+      </Paragraph>
+    ),
+    detectTokens: {
+      anyOf: [
+        'nl.paragraph.lead.color',
+        'nl.paragraph.lead.font-size',
+        'nl.paragraph.lead.font-weight',
+        'nl.paragraph.lead.line-height',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-skip-link',
+    name: 'Candidate Skip Link',
+    render: () => <SkipLink href="main" />,
+    detectTokens: {
+      anyOf: [],
```

Comment:
Ik zou hier wel wat skip-link tokens verwachten. Nu krijgt ieder thema deze story, ongeacht of ze skip-link hebben ingesteld. Je kunt hier kijken welke er zijn https://github.com/nl-design-system/candidate/blob/main/packages/tokens/skip-link-tokens/tokens.json